.circleci/config.yml

```diff
@@ -643,16 +643,7 @@ jobs:
       - DEPLOY_DOWNTIME: 1800 #30M
     steps: *deploy_steps
 
-  upload_packages_osx:
-    <<: *infrastructure_config
-    steps:
-      - checkout
-      - attach_workspace:
-          at: *packages_workspace
-      - *setup_secrets
-      - *upload_github_release_asset
-
-  upload_packages_linux:
+  upload_release_packages:
```

Comment:
I see step `upload_github_release_asset` has `ASSETS=${PACKAGES_DIR:?}/*` so that is ok.